bciers/libs/shared/components/src/lib/form/SingleStepTaskListForm.tsx

```diff
@@ -131,6 +134,7 @@ const SingleStepTaskListForm = ({
           onSubmit={submitHandler}
           liveValidate={isLiveValidate}
         >
+          {error && <Alert severity="error">{error}</Alert>}
```

Comment:
```suggestion
          <div className="min-h-6">
            {error && <Alert severity="error">{error}</Alert>}
          </div>
```

This should be wrapped in a div that is always there with a minimum height set, so that it doesn't jump. I noticed that the read only form is causing it to jump on submit as well.

There is something going on with the read only template as which is causing it to look buggy on submit when it goes readonly for a second. If you can't figure that out I can take a closer look.

Reply:
Okay it seems like ReadOnly widgets need a min height set. They don't jump when some form data exists but when the form is empty they do.... I will do that in another PR myself thanks.